docs/resiliency/disaster-recovery-azure-applications.md

```diff
@@ -8,19 +8,19 @@ ms.date: 08/18/2016
+As with availability considerations, Azure provides [resiliency technical guidance](./index.md) designed to support disaster recovery. There is also a relationship between availability features of Azure and disaster recovery. For example, the management of roles across fault domains increases the availability of an application. Without that management, an unhandled hardware failure would become a “disaster” scenario. Leveraging these availability features and strategies is an important part of disaster-proofing your application. However, this article goes beyond general availability issues to more serious (and rarer) disaster events.
 
 ## Multiple datacenter regions
-Azure maintains datacenters in many regions around the world. This infrastructure supports several disaster recovery scenarios, such as the system-provided geo-replication of Azure Storage to secondary regions. It also means that you can easily and inexpensively deploy a cloud service to multiple locations around the world. Compare this with the cost and difficulty of running your own datacenters in multiple regions. Deploying data and services to multiple regions helps protect your application from major outages in a single region.
+Azure maintains datacenters in many regions around the world. This infrastructure supports several disaster recovery scenarios, such as system-provided geo-replication of Azure Storage to secondary regions. You can also easily and inexpensively deploy a cloud service to multiple locations around the world. Compare this with the cost and difficulty of building and maintaining your own datacenters in multiple regions. Deploying data and services to multiple regions helps protect your application from a major outage in a single region.
 
 ## Azure Traffic Manager
-When a region-specific failure occurs, you must redirect traffic to services or deployments in another region. You can do this routing manually, but it's more efficient to use an automated process. Azure Traffic Manager is designed for this task. You can use it to automatically manage the failover of user traffic to another region in case the primary region fails. Because traffic management is an important part of the overall strategy, it's important to understand the basics of Traffic Manager.
+When a region-specific failure occurs, you must redirect traffic to services or deployments in another region. It is most effective to handle this automatically via tools such as Azure Traffic Manager, which can automatically manage the failover of user traffic to another region in case the primary region fails. Because traffic distribution is an important part of a DR strategy, it is important to understand the basics of Traffic Manager.
```

Comment:
- Agree "effective" not "efficient". 
- TM is better described as a service, rather than a tool. 
- Contractions (it's) are OK and actual encouraged when natural